jerry-core/ecma/builtin-objects/ecma-builtin-global.c

```diff
@@ -302,180 +142,10 @@ ecma_builtin_global_object_parse_int (const lit_utf8_byte_t *string_buff, /**< r
 static ecma_value_t
 ecma_builtin_global_object_parse_float (const lit_utf8_byte_t *string_buff, /**< routine's first argument's
                                                                              *   string buffer */
-                                        lit_utf8_size_t string_buff_size) /**< routine's first argument's
+                                           lit_utf8_size_t string_buff_size) /**< routine's first argument's
```

Comment:
`ecma_builtin_global_object_parse_float`  is only a jumber function now, so it can be removed and `ecma_number_parse_float ` can be called directly. (same for parseInt)